extensions/wikia/UserLogin/FacebookSignupController.class.php

```diff
@@ -115,14 +113,6 @@ public function modal() {
 		// The new FB SDK doesn't define contact_email
 		if ( $this->wg->EnableFacebookClientExt ) {
```

Comment:
Again, should this be part of the fix for [UC-209](https://wikia-inc.atlassian.net/browse/UC-209)?